api/controllers/loginUser.py

```diff
@@ -26,8 +26,10 @@ def login():
             return ErrorResponse(UserNotFound(uid).message, 422, {'Content-Type': 'application/json'}).respond()
 
         tokenObj = {'user': user.username}
-        if user.siteAdmin or user.email in admins:
-            tokenObj = {'adminStatus': True}
+        perm = Permissions.get_by_uid(user.id)
+        if perm:
```

Comment:
i guess this can be removed since you're checking `perm.isAdmin` as well?

Reply:
This is because in case a user tries to get admin token with any user id. In case permissions are not able to find them `perm.isAdmin` will throw exception.

Reply:
cool then!